docs/index.rst

```diff
@@ -40,6 +40,10 @@ This second page allows for the setting of the configuration within the OPC UA s
     It can occur that the Asset Name in an incoming Reading does not reflect the owning object and that the proper name will be found in the hierarchy definition.
     If this is the case, uncheck this box.
 
+  - **Parse Hierarchy from Asset Name**: The plugin can parse a slash-separated path string to create an OPC UA Address Space hierarchy (see :ref:`Parsing of Full Paths`).
```

Comment:
Wrong reference label is being used here. See https://github.com/fledge-iot/fledge-north-opcua/pull/48